src/app/components/cinema-component/cinema-component.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="header">
+  <ng-content></ng-content>
+</div>
+
+<div *ngIf="!isNoMovies; else noMoviesContent">
+  @for (movieList of movieLists; track $index) {
```

Comment:
@for (movie of movieList; track movie.id) {

Reply:
done